build/archives/BUILD.gn

```diff
@@ -172,17 +172,24 @@ if (build_engine_artifacts && !flutter_prebuilt_dart_sdk) {
   zip_bundle_from_file("dart_sdk_archive") {
     deps = [ "//third_party/dart:create_sdk" ]
     output = "dart-sdk-$full_target_platform_name.zip"
-    if (is_mac) {
-      # Mac artifacts sometimes use mac and sometimes darwin. Standardizing the
-      # names will require changes in the list of artifacts the tool is downloading.
-      output = "dart-sdk-darwin-$target_cpu.zip"
-    }
     files = [
       {
         source = rebase_path("$root_build_dir/dart-sdk")
         destination = "dart-sdk"
       },
     ]
+    if (is_mac) {
+      # Mac artifacts sometimes use mac and sometimes darwin. Standardizing the
+      # names will require changes in the list of artifacts the tool is downloading.
```

Comment:
It sounds like this comment should be a TODO with a link to a GitHub issue.

Reply:
Umm I am not familiar with the context of this part, I just copied this paragraph.  Maybe the author @godofredoc  could help take a look, thank you